src/ai/backend/client/func/scaling_group.py

```diff
@@ -123,6 +127,8 @@ async def create(
         scheduler: str = None,
         scheduler_opts: Mapping[str, str] = None,
         use_host_network: bool = False,
+        wsproxy_addr: str = None,
+        wsproxy_api_token: str = None,
```

Comment:
These fields allow `None` as a valid value so the type annotation should be `Optional`.
```suggestion
        wsproxy_addr: Optional[str] = None,
        wsproxy_api_token: Optional[str] = None,
```

Reply:
FYI: All new codes should explicitly define `| None` or `Optional[]` when annotating nullable values in function arguments.
See https://adamj.eu/tech/2022/10/18/python-type-hints-implicit-optional-types/